### PR TITLE
BH-64754: Add for new novo version

### PR DESCRIPTION
**Pull request by kvnfrederiksen**
## **Description**

This will make sure the hotfix applied in 3.16.1 will also publish to the new version of novo in the 3.17.0 release.

#### **Verify that...**

- [x] Any related demos were added and `npm start` and `npm run build` still works
- [x] New demos work in `Safari`, `Chrome` and `Firefox`
- [x] `npm run lint` passes
- [x] `npm test` passes and code coverage is increased
- [x] `npm run build` still works

#### **Bullhorn Internal Developers**
- [x] Run `Novo Automation`
- [x] Run `BBO Automation`

### CODE REVIEW

File: projects/novo-elements/src/elements/data-table/data-table.pipes.ts [hunk 1]
```diff
@@ -46,7 +46,8 @@ export class DateTableDateTimeRendererPipe<T> implements PipeTransform {
   constructor(private labels: NovoLabelService) {}
   transform(value: any, column: IDataTableColumn<T>): string {
     if (!Helpers.isEmpty(value)) {
-      return column.format ? value : this.labels.formatDate(interpolateCell<T>(value, column));
+      let val = interpolateCell<T>(value, column);
+      return this.labels.formatDateShort(val);
     }
```
Comment (amrutha-rajiv) [inline projects/novo-elements/src/elements/data-table/data-table.pipes.ts]: please add unit tests

Comment (kvnfrederiksen) [reply projects/novo-elements/src/elements/data-table/data-table.pipes.ts]: Done!

File: projects/novo-elements/src/elements/data-table/data-table.pipes.ts [hunk 2]
```diff
@@ -46,7 +46,8 @@ export class DateTableDateTimeRendererPipe<T> implements PipeTransform {
   constructor(private labels: NovoLabelService) {}
   transform(value: any, column: IDataTableColumn<T>): string {
     if (!Helpers.isEmpty(value)) {
-      return column.format ? value : this.labels.formatDate(interpolateCell<T>(value, column));
+      let val = interpolateCell<T>(value, column);
```
Comment (tbo47) [inline projects/novo-elements/src/elements/data-table/data-table.pipes.ts]: Can you use const instead of let here?

Comment (kvnfrederiksen) [reply projects/novo-elements/src/elements/data-table/data-table.pipes.ts]: That's a possibility. However, this is for a hotfix, and is a simple reversion to what was there before. As such, I'd rather play it safe with what I know works and move forward quickly. A follow-on ticket will be set up to address concerns such as these.

**Pull request merged**